### ISSUE TITLE
Nimble build fails using nim devel using the recent `alignas` change

### ISSUE THREAD

**Issue by kaushalmodi** [nim-lang/Nim#12661]
Building of nimble using a recent build of nim from devel branch fails.

> Error: unhandled exception: 'alignment' is not accessible using discriminant 'kind' of type 'TSym' [FieldError]

### Current Output
```
bin/nim c -o:bin/nimble --noNimblePath --nilseqs:on -d:release  dist/nimble/src/nimble.nim

Hint: used config file '/home/kmodi/downloads/git/Nim/config/nim.cfg' [Conf]
Hint: used config file '/home/kmodi/downloads/git/Nim/dist/nimble/src/nimble.nim.cfg' [Conf]
Hint: used config file '/home/kmodi/downloads/git/Nim/config/config.nims' [Conf]
Hint: used config file '/home/kmodi/.config/nim/config.nims' [Conf]
fatal.nim(39)            sysFatal
Error: unhandled exception: 'alignment' is not accessible using discriminant 'kind' of type 'TSym' [FieldError]
FAILURE
```

### Expected Output

`nimble` should build fine.

### Additional Information

- I found this from a Travis cron of one of my projects: https://travis-ci.org/OrgTangle/ntangle/builds/611959593
- Attempting to build nimble with any nim devel version after https://github.com/nim-lang/Nim/commit/0496a666e22465eba9a309f0974220988f7f9920 shows this issue.

/cc @krux02

**Comment by a-mr** [comment]
FWIW For me compilation failed on Windows with this error, but succeeded on Linux, strange.

**Comment by Araq** [comment]
Fixed it already, I think.

**Comment by kaushalmodi** [comment]
Thanks, I confirm the fix on 64-bit Linux RHEL 6.8.
